libnmstate/nm/connection.py

```diff
@@ -333,6 +345,15 @@ def _activation_progress_check(self, action):
                 NmstateLibnmError(f"{action} failed: reason={reason}")
             )
 
+    def _is_sriov_parameter_not_supported_by_driver(self):
+        if self.nmdevice:
+            return (
+                self.nmdevice.props.state_reason
+                == NM.DeviceStateReason.SRIOV_CONFIGURATION_FAILED
```

Comment:
`return self.nmdevice and self.nmdevice.props.state_reason == NM.DeviceStateReason.SRIOV_CONFIGURATION_FAILED`

Reply:
Done, thanks!